src/main/scripts/ctl/wmtsServerGetFeatureInfoTests.xml

```diff
@@ -946,7 +946,7 @@ the request entity is a valid KVP GET GetFeatureInfo request with an optional sa
 	14.  the request entity is a valid KVP GET GetFeatureInfo request with the mandatory I parameter with a column index value from zero to the value of &lt;TileWidth&gt; minus 1 for the specified TileMatrix from the Service Metadata document;	
 	15.  the request entity is a valid KVP GET GetFeatureInfo request with the mandatory infoFormat parameter with an InfoFormat value that is not one of the &lt;InfoFormat&gt;s for the specified Layer from the Service Metadata document;
 	16.  the response entity is a schema-valid ows:Exception document.
-	17.  the response entity contains an exceptionCode attribute value of "InvalidParameterValue".
+	17.  the response entity contains an exceptionCode attribute value of "PointIJOutOfRange".
```

Comment:
Shouldn't this be changed in test `wmts:Server.KVP.GET.GetFeatureInfo.Invalid.Pixel.Col.I`?

Reply:
You are indeed right, commit fixed.